discord/ui/input_text.py

```diff
@@ -138,7 +152,9 @@ def min_length(self) -> Optional[int]:
     @min_length.setter
     def min_length(self, value: Optional[int]):
         if value and not isinstance(value, int):
-            raise TypeError(f"min_length must be None or int not {value.__class__}")  # type: ignore
+            raise TypeError(f"min_length must be None or int not {value.__class__.__name__}")  # type: ignore
+        if 0 <= value <= 4000:
+            raise ValueError("min_length must be between 0 and 4000")
```

Comment:
Wouldn't `value` already be between 0 and 4000 in this case?

Reply:
Awkwaaaaard. Updated to the correct logic there and in `max_length.setter`